hack/test_e2e.sh

```diff
@@ -39,6 +39,159 @@ GINKGO_DUMP_LOGS_ON_FAILURE=false
 GINKGO_DISABLE_LOG_DUMP=true
 GINKGO_PROVIDER="local"
 
+# iptables specific skipped ginkgo tests
```

Comment:
I would recommend putting all these var names in their own file titled like `test_skip_lists.sh` which you could just source, also make each var a single assignment like 

```bash
## Iptables, ipv4
GINKGO_SKIP_ipv4_iptables_TEST="should be updated after adding or deleting ports\n
|should serve multiport endpoints from pods\n
|should check kube-proxy urls\n
etc etc "

GINKGO_SKIP_ipv6_iptables_TEST=...
```

I think that may be more readable?

Reply:
I agree with moving them out into a separate file.
But I find it easier to keep each filter on a separate row.
I have been running the tests over and over and over again to find all filters that must be updated and I found it easier to use row for every filter when adding in the new ones.

This should after all be a temp solution. The next step must be to fix the backends in order to get rid of these skip filters

Reply:
Hmm

What you propose is not \n at the end of each row right, bit a single \
That should be even easier to manage then what is there now